django/contrib/auth/tests/custom_user.py

```diff
@@ -221,6 +224,31 @@ class CustomUserBadRequiredFields(AbstractBaseUser):
     class Meta:
         app_label = 'auth'
 
+
+class CustomUserUUIDasPK(AbstractBaseUser, PermissionsMixin):
```

Comment:
Do you check if there is a way to use other models?
It will be good to inherit one and add only needed field that reproduces your changes